templates/explore/search.tmpl

```diff
@@ -18,7 +18,7 @@
 <form class="ui form ignore-dirty" style="max-width: 90%">
 	<input type="hidden" name="tab" value="{{$.TabName}}">
 	<div class="ui fluid action input">
-	  <input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}..." autofocus>
+	  <input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}...">
```

Comment:
The same as the above.

Reply:
This sub-template is used on the issue list where I do want to remove it.